database/db.js

```diff
@@ -12,21 +12,54 @@ db.once('open', function() {
   console.log("connected to the DB: ")
   });
 
-
-
-  
-
-
-const imagemodal = new Schema({
+const imageModal = new Schema({
   restaurauntTitle: String,
   helpfull: Number,
   notHelpfull: Number,
   images: Array
-
 });
 
-//use for reference in seeding script
+const getImages = () => {
+  imageModal.find()
+    .then(console.log('found images'))
+    .catch(console.log('cant find images'))
+}
+
+const getImageById = (id) => {
+  imageModal.findById(id)
+    .then(console.log(`found image by ${id}`))
+    .catch(console.log(`couldn't find image by ${id}`))
+}
 
+const postImage = (image) => {
+  image.save()
+    .then(console.log('image saved to database'))
+    .catch(console.log('image not saved by database'))
+}
+
+const updateImage = (id,image) => {
+  imageModal.update({_id: id},{image}, {upsert:true})
+    .then(console.log(`update image at id: ${id}`))
+    .catch(console.log(`failed to update image at id ${id}`))
+}
+
+const partialUpdate = (id,image) => {
+  imageModal.update({_id: id},{$set: image}, {upsert: true})
+    .then(console.log(`partial update at id: ${id}`))
+    .catch(console.log(`failed to update image at id ${id}`))
+}
+
+const deleteImage = (id) => {
+  imageModal.findAndModify(_id: id)
+    .then(console.log(`delete image at id: ${id}`))
+    .catch(console.log(`failed to delete item with id ${id}`))
+}
+
+const deleteAllImages = () => {
+  imageModal.remove()
+    .then(console.log('deleted all images'))
+    .catch(console.log('failed to delete all images'))
+}
 
```

Comment:
send back success/error messages to the server that made the query request. Logging to the console will not give the server/user feedback if the request was successful or not

Reply:
got it, yea I was thinking of setting up these logs to catch errors when I test with postman later